tests/data/references/multiple-gateways.yaml

```diff
@@ -1,60 +1,60 @@
 kind: VirtualService
-apiVersion: networking.istio.io/v1beta1
+apiVersion: networking.istio.io/v1
 metadata:
   name: reviews-well
   namespace: bookinfo
 spec:
   hosts:
-    - 'reviews.prod.svc.cluster.local'
+  - 'reviews.prod.svc.cluster.local'
   gateways:
-    - gateway1
-    - bookinfo2/gateway2
-    - mesh
```

Comment:
Some yaml linter?

Reply:
Yes, we added yaml linter some time ago (https://github.com/kiali/kiali/pull/7011) to ensure consistency across Kiali application.